### DISCUSSION TITLE
OpenMMLab 2.0公测之 MMCV 2.x 中文讨论区

### DISCUSSION THREAD

**Discussion by vansin** [open-mmlab/mmcv#2258] [General]
欢迎加入 OpenMMLab 2.0公测之 mmcv 2.x 中文讨论区

**Comment by deel789** [comment]
ModuleNotFoundError: No module named '__main__.version'; '__main__' is not a package

**Reply by deel789** [reply]
出现这样的错误如何调整？

**Reply by deel789** [reply]
f'MMCV=={mmcv.__version__} is used but incompatible. ' \
AssertionError: MMCV==2.0.0rc1 is used but incompatible. Please install mmcv>=1.4.2, <=1.7.0.

以及这样的版本要求的错误，怎么调整呢？

**Reply by vansin** [reply]
OpenMMLab 1.x 和 2.x 的算法库不能混用哈~

**Reply by vansin** [reply]
先介绍一些用了哪些OpenMMLab 的库，是什么版本的？

**Comment by sidiangongyuan** [comment]
我在导入
from mmcv.cnn import xavier_init, constant_init
from mmcv.runner import force_fp32, auto_fp16
from mmcv.runner.base_module import BaseModule, ModuleList, Sequential
中，分别在xavier_init, constant_init runner下出现红色波浪线，错误是 cannot fine the reference 。 什么原因呢
